### ISSUE TITLE
How does openbazaar look for GPG4Win?

### ISSUE THREAD

**Issue by greatwolf** [OpenBazaar/OpenBazaar#1203]
I installed GPG on Windows 7 previously and openbazaar was able to detect it. But for whatever reason, openbazaar fails with this message whenever I start it up:

![image](https://cloud.githubusercontent.com/assets/1663586/6834332/a2170d12-d2f0-11e4-8034-63a96243efa2.png)

I have not changed anything on my machine's setup.

How exactly is openbazaar trying to detect GPG4Win? Is it searching for some environment variable? Note I manually re-added my installed GPG path to my environment path variable. I recall it being there on first install but I no longer see it there for some reason. This does not fix the detection issue with openbazaar however.

**Comment by greatwolf** [comment]
Okay, found the problem. Checking the source here 

https://github.com/OpenBazaar/OpenBazaar/blob/0c65fbd93b0a2f185af5528761ab7f51909ddcc8/installers/windows/openbazaar.py#L36

shows it's using python's [`find_library`](https://docs.python.org/2/library/ctypes.html#finding-shared-libraries) function which does indeed look under the `PATH` environment for the current shell session under Windows.

The problem was that I quotatifed the GPG directory when appending to my `PATH`. Turns out this is unnecessary for `PATH` appending, even if there are spaces in the fully qualified path that's being added.
